draft-bbs-signatures.md

```diff
@@ -237,9 +237,13 @@ Throughout the operations of this signature scheme, each message that is signed
 
 The KeyGen algorithm generates a secret key SK deterministically from a secret octet string IKM.
 
+**Note** Implementing this operation as defined below for a particular ciphersuite is RECOMMENDED,
+however implementers are able to define an alternative key generation procedure provided it outputs
+a secret key of the equivalent form.
```

Comment:
Maybe we can tighten up 'of the equivalent form'. I think the main requirement is that it chooses keys uniformly from the key space (the scalar field).

Reply:
Agreed. Maybe "...provided that the secret key is statistically close to uniformly random" (taken from the [bls_spec](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-04#section-2.3)) or "...provided that the secret key is infeasible to guess"?? I think the second one covers most security requirements (in the first wording one may wonder how close to uniform is enough, although maybe I'm overthinking it)??